data/json/recipes/food/distill.json

```diff
@@ -8,8 +8,8 @@
     "difficulty": 3,
     "time": "30 m",
     "batch_time_factors": [ 50, 4 ],
-    "autolearn": [ [ "cooking", 4 ] ],
-    "book_learn": [ [ "brewing_cookbook", 2 ] ],
+    "autolearn": [ [ "cooking", 8 ] ],
+    "book_learn": [ [ "brewing_cookbook", 2 ], [ "winemaking_beginner", 2 ] ],
```

Comment:
This shouldn't be in either of those books. We should add a distilling book.

Reply:
done.  Though I left brandy in winemaking at the moment since I thought brandy was fortified wine?